tests/func/experiments/test_remove.py

```diff
@@ -121,6 +122,11 @@ def test_remove_remote(tmp_dir, scm, dvc, exp_stage, git_upstream, use_url):
     assert git_upstream.tmp_dir.scm.get_ref(str(ref_info_list[1])) is None
     assert git_upstream.tmp_dir.scm.get_ref(str(ref_info_list[2])) == exp_list[2]
 
+    with pytest.raises(
```

Comment:
@dberenbaum Added a test myself because it was just quicker

Reply:
Sorry, should have done this myself. Thank you though!